challenges-webapp/src/main/webapp/vue-app/engagement-center/components/EngagementCenter.vue

```diff
@@ -54,6 +54,12 @@ export default {
   data: () => ({
     engagementCenterEnabled: eXo.env.portal.engagementCenterEnabled,
     tab: null,
+    earnerId: -1,
   }),
+  created() {
+    if (!this.isAdministrator) {
+      this.earnerId = eXo.env.portal.userIdentityId;
+    }
+  },
```

Comment:
Make `earnerId` initialized to current user independantly from his role.

```suggestion
    earnerId: eXo.env.portal.userIdentityId,
  }),
```